src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -2008,28 +2133,42 @@ public PreSubmitCallbackHandler<AsylumCase> ftpaApplicationDecisionGrantedOrPart
-                    .orElse(false);
 
-                if (!isGrantedOrPartiallyGrantedOutcome) {
-                    isGrantedOrPartiallyGrantedOutcome = asylumCase
-                        .read(AsylumCaseDefinition.FTPA_APPELLANT_RJ_DECISION_OUTCOME_TYPE,
-                            FtpaDecisionOutcomeType.class)
-                        .map(decision -> decision.toString().equals(FtpaDecisionOutcomeType.FTPA_GRANTED.toString())
-                                         || decision.toString().equals(FtpaDecisionOutcomeType.FTPA_PARTIALLY_GRANTED.toString()))
-                        .orElse(false);
-                }
+                return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
+                       && (callback.getEvent() == Event.LEADERSHIP_JUDGE_FTPA_DECISION
+                           || callback.getEvent() == Event.RESIDENT_JUDGE_FTPA_DECISION)
+                       && isGrantedOrPartiallyGrantedOutcome(asylumCase,
+                    FTPA_APPELLANT_DECISION_OUTCOME_TYPE, FTPA_APPELLANT_RJ_DECISION_OUTCOME_TYPE)
+                       && !hasThisNotificationSentBefore(asylumCase, callback,
+                    "_FTPA_APPLICATION_DECISION_LEGAL_REPRESENTATIVE_APPELLANT")
+                       && !isAipJourney(asylumCase);
+            },
+            notificationGenerators
+        );
+    }
+
+    @Bean
+    public PreSubmitCallbackHandler<AsylumCase> ftpaApplicationDecisionGrantedOrPartiallyGrantedAppellantAipJourneyNotificationHandler(
+        @Qualifier("ftpaApplicationDecisionGrantedOrPartiallyGrantedAppellantAipJourneyNotificationGenerator")
+            List<NotificationGenerator> notificationGenerators) {
+
+        // RIA-6113
+        return new NotificationHandler(
+            (callbackStage, callback) -> {
+
+                AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
 
                 return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
                        && (callback.getEvent() == Event.LEADERSHIP_JUDGE_FTPA_DECISION
                            || callback.getEvent() == Event.RESIDENT_JUDGE_FTPA_DECISION)
-                       && !AsylumCaseUtils.isInternalCase(asylumCase)
-                       && isGrantedOrPartiallyGrantedOutcome
+                       && isGrantedOrPartiallyGrantedOutcome(asylumCase,
+                    FTPA_APPELLANT_DECISION_OUTCOME_TYPE, FTPA_APPELLANT_RJ_DECISION_OUTCOME_TYPE)
                        && !hasThisNotificationSentBefore(asylumCase, callback,
-                    "_FTPA_APPLICATION_DECISION_LEGAL_REPRESENTATIVE_APPELLANT");
+                    "_FTPA_APPLICATION_DECISION_TO_APPELLANT_EMAIL")
+                       && !hasThisNotificationSentBefore(asylumCase, callback,
+                    "_FTPA_APPLICATION_DECISION_TO_APPELLANT_SMS")
+                       && isAipJourney(asylumCase)
+                       && !AsylumCaseUtils.isInternalCase(asylumCase);
```

Comment:
I dont think internal case check is needed here.

Reply:
removed check